bcipy/tasks/tests/test_decision_maker.py

```diff
@@ -0,0 +1,111 @@
+"""Tests for Copy Phrase DecisionMaker"""
+
+import unittest
+import numpy as np
+from mockito import any, mock, unstub, when
+
+import bcipy.tasks.rsvp.main_frame as mf
+from bcipy.helpers.load import load_json_parameters
+
+
+class TestDecisionMaker(unittest.TestCase):
+    """Tests for DecisionMaker"""
+
+    def test_decision_maker(self):
+        """Test default behavior"""
+        decision_maker = mf.DecisionMaker(
+            min_num_seq=1,
+            max_num_seq=10,
+            state='',
+            alphabet=['a', 'b', 'c', 'd'],
+            decision_threshold=0.8)
+
+        likelihood = np.array([0.2, 0.2, 0.2, 0.4])
+        decision, _arg = decision_maker.decide(likelihood)
+        self.assertFalse(decision)
+        self.assertEqual('.', decision_maker.state)
+        self.assertEqual('', decision_maker.displayed_state)
+
+        likelihood = np.array([0.05, 0.05, 0.1, 0.8])
+        decision, _arg = decision_maker.decide(likelihood)
+        self.assertFalse(decision, "Must exceed threshold")
+        self.assertEqual('..', decision_maker.state)
+
+        likelihood = np.array([0.05, 0.05, 0.09, 0.81])
+        decision, _arg = decision_maker.decide(likelihood)
+        self.assertTrue(decision)
+        self.assertEqual('..d', decision_maker.state)
+        self.assertEqual('d', decision_maker.displayed_state)
+
+    def test_min_sequences(self):
+        """Test min sequences criteria."""
+
+        decision_maker = mf.DecisionMaker(
+            min_num_seq=2,
+            max_num_seq=10,
+            state='',
+            alphabet=['a', 'b', 'c', 'd'],
+            decision_threshold=0.8)
+
+        likelihood = np.array([0.05, 0.05, 0.09, 0.81])
```

Comment:
This test case might benefit from a setup method. We could pull out a default likelihood, decision maker etc which the unit tests could alter if needed.